services/common/boltzmann/client/index.js

```diff
@@ -0,0 +1,207 @@
+'use strict'
+
+const fetch = require('node-fetch')
+const os = require('os')
+
+const hostname = os.hostname()
+
+const e = encodeURIComponent
```

Comment:
What do you think of wrapping this in a function? If any future refactors need to happen it could make it easier.

`const encode = (str) => encodeURIComponent(str)`